core/kazoo_stdlib/test/kz_json_test.erl

```diff
@@ -19,21 +19,28 @@
 %% PropEr Testing
 -ifdef(PROPER).
 
-is_json_object_proper_test_() ->
+%% Lifted from Erlang ML
+proper_test_() ->
     {"Runs kz_json PropEr tests"
     ,{'timeout'
      ,10000
-     ,[?_assertEqual([], proper:module(?MODULE, [{'to_file', 'user'}
-                                                ,{'numtests', 500}
-                                                ]
-                                      ))
+     ,[{atom_to_list(F)
+       ,fun () ->
+                ?assert(proper:quickcheck(?MODULE:F(), [{'to_file', 'user'}
+                                                       ,{'numtests', 500}
+                                                       ]))
+        end
+       }
+       || {F, 0} <- ?MODULE:module_info('exports'),
+          F > 'prop_',
+          F < 'prop`'
```

Comment:
Pretty sure this is not doing what you think it's doing: needs conversion to a string type.